CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
+
+## [1.0-beta-1] - 2018-12-XX
+- [[JENKINS-54375](https://issues.jenkins-ci.org/browse/JENKINS-54375)] First release
```

Comment:
It would be nice to have a changelog since the commit used in https://github.com/jenkins-x/jenkins-x-serverless/blob/master/packager-config.yml#L19 . We could tag that version as `alpha-1`, for example

It would help the Jenkins X team to review the changes and potential impact

Reply:
Done!